__tests__/app.test.js

```diff
@@ -112,3 +112,53 @@ describe("GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: returns array of comments associated with valid specified review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(3);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: 2,
+          });
+        });
+      });
+  });
+  test("200: returns empty array when given a review_id that exists but has no associated comments", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(response.body).toHaveProperty("comments");
+        expect(comments.length).toBe(0);
+      });
+  });
+  test("404: returns an error when given review_id that does not exist", () => {
+    return request(app)
+      .get("/api/reviews/150/comments")
+      .expect(404)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("There is no user with that id");
+      });
+  });
+  test("400: returns an error when review_id is not in the correct format", () => {
```

Comment:
If I was being very picky... 
I would say that this test is checking similar behaviour (albeit on a different endpoint) to your test on line 150. 
`400: returns 400 when given anything other than a number` but the descriptions are very different in language and tone. 
It can often be neater to aim for a little more consistency in the style of your test descriptions.